src/components/ListBooks.js

```diff
@@ -0,0 +1,42 @@
+const ListBooks = (props) => {
+  const { books } = props;
+
+  return (
+    <div className="list">
+      {
+          books.map((book, index) => (
+            <div key={book.title} className="list-item" id={index}>
+              <div className="left">
+                <strong>{book.category}</strong>
+                <h2>{book.title}</h2>
+                <p>{book.author}</p>
+                <div className="interact">
+                  <span>Comments</span>
+                  |
+                  <span>Remove</span>
```

Comment:
- [ ] Please make this a button to remove books to comply with the project requirements.